crowbar_framework/app/models/api/crowbar.rb

```diff
@@ -125,5 +137,11 @@ def addon_installed?(addon)
     rescue NameError
       false
     end
+
+    def repo_version_available(product, version)
+      `zypper products`.split("\n").detect do |p|
+        p =~ /#{product}.*#{version}-.*/
```

Comment:
I think the regex is still not ideal... would be good to match the values to columns. unfortunately zypper doesn't provide any easy to parse output format.